### PR TITLE
Improve homepage copy

### PR DESCRIPTION
**Pull request by alexleventer**
Improve CassIO website homepage copy

### CODE REVIEW

**Comment by alexleventer** [comment]
Fixes #24

File: docs/index.md [hunk 1]
```diff
@@ -3,7 +3,13 @@
 <!-- ![CassIO logo](images/cassio_logo1_transparent.png#only-light) -->
 <!-- ![CassIO logo](images/cassio_logo1.png#only-dark) -->
 
-CassIO is the ultimate solution for seamlessly integrating [Apache Cassandra®](https://cassandra.apache.org) with generative artificial intelligence and other machine learning workloads. This powerful Python library simplifies the complicated process of accessing the advanced features of the Cassandra database, including vector search capabilities. With CassIO, developers can fully concentrate on designing and perfecting their AI systems without any concerns regarding the complexities of integration with Cassandra.
+CassIO is a python library developed for integrating Apache Cassandra® with generative artificial intelligence (AI) and other machine learning workloads.
```
Comment (hemidactylus) [inline docs/index.md:6]: I'd capitalize "Python" preferrably.

File: docs/index.md [hunk 2]
```diff
@@ -3,7 +3,13 @@
...
+
+CassIO abstracts the process of accessing Cassandra, including Vector Search capabilities, offering a set of ready-to-use tools that minimize the need for additional code.
+
+CassIO is framework agnostic, streamlining integration irrespective of the AI toolkit used, and also integrates seamlessly with LangChain. It provides various features, including memory modules for storing conversational data, caching responses, automatic data injection, and support for "partialing" of prompts.
+
```
Comment (hemidactylus) [inline docs/index.md:11]: "and also integrates seamlessly with LangChain" seems a repetition to me, once it's already said "streamlining integration irrespective of the AI toolkit used".

File: docs/index.md [hunk 3]
```diff
@@ -3,7 +3,13 @@
...
+CassIO leverages Vector Search capabilities in Cassandra and DataStax Astra DB to enable "semantically aware" tooling and scalable, low-latency access to data, including a cache of language model responses, a "semantic index" for knowledge storage and retrieval, and a "semantic memory" for chat interactions
```
Comment (hemidactylus) [inline docs/index.md:12]: missing a full stop at the end.

Comment (hemidactylus) [reply docs/index.md:12]: @alexleventer could you please address the above?

**Comment by hemidactylus** [comment]
Closing this for now for lack of activity.
The appearance of the site has changed enough for this issue to be a bit stale.

**Pull request closed without merging**